### PR TITLE
Update workshop.md

### PR DESCRIPTION
**Pull request by idomic**
Some sample PR

### CODE REVIEW

File: material/tasks/clean.py [hunk 1]
```diff
@@ -0,0 +1,51 @@
...
+product = None
+
+# %% tags=["soorgeon-unpickle"]
+df = pd.read_parquet(upstream['load']['df'])
+
+# %% [markdown]
+# ## Clean
+
+# %%
+sns.histplot(df.HouseAge)
```
Comment (edublancas) [inline material/tasks/clean.py:28]: hey good job

File: material/tasks/clean.py [hunk 2]
```diff
@@ -0,0 +1,51 @@
...
+
+# %%
+sns.histplot(x=df.AveBedrms)
+
+# %%
+sns.boxplot(x=df.AveBedrms)
+
+# %%
+# let's also remove big houses
+df = df[df.AveBedrms <= 4]
```
Comment (edublancas) [inline material/tasks/clean.py:43]: why 4? maybe 4.4?

File: material/tasks/linear-regression.py [hunk 3]
```diff
@@ -0,0 +1,44 @@
...
+
+# %% tags=["parameters"]
+upstream = ['train-test-split']
+product = None
+
+# %% tags=["soorgeon-unpickle"]
+X_test = pickle.loads(Path(upstream['train-test-split']['X_test']).read_bytes())
+X_train = pickle.loads(Path(upstream['train-test-split']['X_train']).read_bytes())
+y_test = pickle.loads(Path(upstream['train-test-split']['y_test']).read_bytes())
+y_train = pickle.loads(Path(upstream['train-test-split']['y_train']).read_bytes())
```
Comment (edublancas) [inline material/tasks/linear-regression.py:26]: i like this

**Review by edublancas — CHANGES_REQUESTED** [review]

File: material/tasks/clean.py (hunk 2 above)
Comment (idomic) [reply material/tasks/clean.py:43]: Working on it!

File: material/tasks/linear-regression.py (hunk 3 above)
Comment (idomic) [reply material/tasks/linear-regression.py:26]: That's awesome!

File: material/tasks/clean.py (hunk 1 above)
Comment (idomic) [reply material/tasks/clean.py:28]: Had a good mentor

**Pull request closed without merging**